c/build_all/linux/build_proton.sh

```diff
@@ -69,8 +69,13 @@ sync_proton ()
     fi
 
     rm $build_root -r -f
-    mkdir $build_root
-    git clone -b $proton_branch $proton_repo $build_root
+    cd /tmp
+    curl -L $proton_repo/archive/$proton_branch.zip -o proton.zip
+    mkdir proton
+    unzip proton.zip -d proton
```

Comment:
This script handles the general case, and it's a convenience for users--it's not a required step to build the project. I don't see a strong reason for changing git to unzip here.  In specialized scenarios the script can be customized; changes wouldn't have to be contributed back.